packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/assert.parsesToRuns.values..st

```diff
@@ -0,0 +1,9 @@
+accessing
+assert: markdownString parsesToRuns: runs values: values
+
+	| parsedText |
+	parsedText := parser parse: markdownString.
+	self
+		assert: markdownString equals: parsedText string;
```

Comment:
Why does this need to be ensured? For now, this is correct, but the method doesn't express this check in the title.

Reply:
I couldn't think of a better name. As our parser returns a `Text`, we want to test against this text, which is made up of a `string` and `runs` which has another time `runs` and `values`. I want to keep all three asserts. But I'll gladly collect naming proposals.